packages/hash/backend-utils/src/postgres.ts

```diff
@@ -21,6 +21,7 @@ export const createPostgresConnPool = (
 
   return createPool(connStr, {
     captureStackTrace: true,
+    connectionTimeout: 10_000,
```

Comment:
Wouldn't this timeout increase only mean that the initial connection timeout is increased, and not necessarily query timeouts? Do you know if some of the other timeouts in the [linked section](https://github.com/gajus/slonik#default-timeouts) would be better suited to tweak for longer queries?

Reply:
It's my understanding that this thing is creating a connection pool, from which individual connections are then created when we want to actually execute a transaction/query. 

There is a `statementTimeout`, but that would be a timeout for an individual statement, whereas a connection might execute several statements within it (if it is opened as a transaction).

I should've put the error message in the description which helps reinforce this:
```
ConnectionError: timeout exceeded when trying to connect\n    at Object.createConnection 
(/app/node_modules/slonik/src/factories/createConnection.ts:99:15)\n    at runNextTicks
```

Reply:
Okay, sounds good :+1: